Source/WebCore/dom/Document.h

```diff
@@ -510,6 +511,7 @@ class Document
     WEBCORE_EXPORT DOMImplementation& implementation();
     
     Element* documentElement() const { return m_documentElement.get(); }
+    AsyncNodeDeletionQueue& nodesToAsyncDelete() { return m_nodesToAsyncDelete; };
```

Comment:
We should probably call this member function: asyncNodeDeletionQueue()
and rename the member variable to m_asyncNodeDeletionQueue as well.